manifests/install.pp

```diff
@@ -26,6 +27,7 @@
     package { 'gitlab-omnibus':
       ensure  => $package_ensure,
       name    => $_package_name,
+      mark    => $package_hold,
```

Comment:
is it safe to always set it, or should the default be undef?

Reply:
@bastelfreak Yes it should. The parameter calls the `mark`parameter on the package resource.

* It only affects Debian and Solaris OS families
* The allowed values are 'hold' or 'none', and the default being 'none'.

https://www.puppet.com/docs/puppet/8/types/package#package-attribute-mark

That being said it is reasonable to set it to `undef`but not sure if the Enum accepts it,  Perhaps making the Enum optional is a solution?